fuel/app/classes/materia/widget/instance.php

```diff
@@ -550,6 +580,63 @@ public function lti_associations()
 			->get();
 	}
 
+	public function get_all_extra_attempts()
```

Comment:
add return type hint

```suggestion
	public function get_all_extra_attempts(): array
```